TwitterStuff

```diff
@@ -0,0 +1,9 @@
+TwitterStuff:
+
+Consumer Key (API Key)	3sqEK5AwQUI32isQAV4LGTdDa
+
+Consumer Secret (API Secret)	9xjKjy5oovPD5fBeH3JwI4gBjPVwSmaqFUAXW0eLEE7A0yvTwZ
+
+Access Token	833009544758444033-K7Kb08YAz5uMHXc2jFzCDeiCq5SJhyQ
+
+Access Token Secret	gPqGBcsY0YDCVY3wcfVIj2DhgV6y3JtRVMRxDQjpr8yNE
```

Comment:
This file should not be here at all. Keep API credentials in a private location. Take a look at in-class note about this.